### ISSUE TITLE
Fix Repos for CI

### ISSUE THREAD

**Issue by ayoung8** [hres/api-lnhpd#8]
Think we can fix: 
prod's repo: api-cnf, api-ctweb, api-dpd, api-mdall, api-noc

Asif you OK with Samir trying? 

api-mdall on dev; api-lnhpd on prod, should be fine. We just have to monitor it when we do an upgrade

Asif you OK with Samir trying? You can do you lnhpd update after to test.

**Comment by ahasangit** [comment]
Yes, please fix all the repos listed.  We will update api-lnhpd afterwards to test.

**Comment by shussain** [comment]
> prod's repo: api-cnf, api-ctweb, api-dpd, api-mdall, api-noc

Fixed the above repo at prod server.

**Comment by ahasangit** [comment]
Thank you Samir.  Would the dev branches for the repos need to be fixed as well?  We will be using these to merge into the prod branches.

**Comment by shussain** [comment]
@ahasangit  I am unsure about your question. My undresanding was that only api-mdall was goign to get a merge (from master to the dev branch).


As an aside, i have tested the existng prod repo that have been fixed.

**Comment by ahasangit** [comment]
@shussain  Our first test was going to be with the api-lnhpd merge from master into the dev branch.  Based on our results from this we can then assess the impact for the other repos.  Most of them will be getting updated by merges in the next week or so as well.

**Comment by ayoung8** [comment]
Ok sounds like a plan
@ahasangit do the pull request and jenkins should pick up the change and we can see if it breaks

**Comment by ahasangit** [comment]
I've merged into the prod branch for lnhpd.  Please confirm @ayoung8 @shussain
